README.md

```diff
@@ -30,6 +30,12 @@ export MANAGEIQ_HOST=localhost
 export MANAGEIQ_PORT=3000
 ```
 
+If the topology inventory requires authentication (ie in dev), basic authentication is supported via these variables. They won't be read in unless :
+```
+export RHN_DEV_USERNAME=myuser
+export RHN_DEV_PASSWORD=password
```

Comment:
Was trying to get this comment in before the names where updated.

I agree with coming up with a common name we can use with other services but suggest we avoid `RHN` in this open project.  I'm open to suggestions here.

Reply:
@gmcculloug We can use 3SCALE or RHN prefix just using the DEV_USERNAME would cause conflicts

Reply:
Plus you would have to be in the VPN to run this

Reply:
Would `CLG_` work?  or `CAT_`?

Reply:
I like what @syncrou suggested, since this is going to be across Catalog (Topology,  approval, RBAC) it would make sense to use something that referred to Catalog in some way. I'm partial to `CLG_`